packages/tools/scripts/generate-packages-json.ts

```diff
@@ -38,7 +38,7 @@ const main = async () => {
 
   await fs.writeFile(
     join(baseDir, 'packages.json'),
-    JSON.stringify(packages, null, 2),
+    JSON.stringify(packages, null, 2) + '\n',
```

Comment:
you can format the file with prettier instead so it solves all possible formatting issues.
```TS
await prettier.format(JSON.stringify(packages, null, 2), { parser: "json" }),
 ```

Reply:
Will try!

Reply:
Confirmed, it also seems to change the order in which the packages are listed, but that's fine.

Reply:
Do you want to update this PR?